frontend/src/components/Main/ModelsForm/ModelsCheckBoxHeader.js

```diff
@@ -15,6 +16,7 @@ const areAllModelsChecked = function(modelType, type, models) {
                 <label className="m-0">
                     <input
                         type="checkbox"
+                        disabled={type === "bayesian" && store.getModelType === "C" ? true : false}
```

Comment:
Instead of this approach, can we make a prop called disabled, with the default set to false? And then we can pass diabled=true for bayesian models instead?